app/src/main/java/com/android/unio/ui/navigation/NavigationAction.kt

```diff
@@ -129,6 +129,7 @@ object Screen {
   const val EXPLORE = "Explore_Screen"
   const val SAVED = "Saved_Screen"
   const val MY_PROFILE = "MyProfile_Screen"
+  const val EDIT_PROFILE = "Edit_Profile_Screen"
   const val SOMEONE_ELSE_PROFILE = "SomeoneElseProfile_Screen"
   const val SETTINGS = "Settings"
   const val ASSOCIATION_PROFILE = "Association_Profile_Screen"
```

Comment:
I just realized, here, we should settle on one format, e.g. `EditProfile_Screen` or `Edit_Profile_Screen`. If you look at the other screens, you can see we defined `SomeoneElseProfile_Screen`, and did not use an `_` for each word separation, only as a prefix for the `Screen` word. This is not your fault, but we should keep only one format.